src/main/kotlin/com/beust/kobalt/misc/CheckVersions.kt

```diff
@@ -20,7 +21,13 @@ public class CheckVersions @Inject constructor(val depFactory : DepFactory,
             arrayListOf(it.compileDependencies, it.testDependencies).forEach { cds ->
                 cds.forEach { compileDependency ->
                     if (MavenId.isMavenId(compileDependency.id)) {
-                        val dep = depFactory.create(compileDependency.shortId, executor, false /* go remote */)
+                        val dep: IClasspathDependency
+                        try {
+                            dep = depFactory.create(compileDependency.shortId, executor, false /* go remote */)
+                        } catch(e: Exception) {
```

Comment:
How about moving everything line 31 and below here? This way the catch only needs to log and do nothing (avoids the `return`.